dockerfiles/diy/dockerfiles/Dockerfile.release

```diff
@@ -3,40 +3,44 @@
 # builds to work on platforms such as Mac OS X/M1 (with some help from
```

Comment:
This file is almost fully identical to the `Dockerfile.router` above, now more than ever.  The only notable difference is that the other file above supports the debug build argument.  We should align these in the future?